src/zeroD/ReactorBase.cpp

```diff
@@ -51,6 +51,20 @@ void ReactorBase::setThermoMgr(ThermoPhase& thermo)
     m_pressure = m_thermo->pressure();
 }
 
+void ReactorBase::setThermoMgr(ThermoPhase& thermo)
+{
+    warn_deprecated("ReactorBase::setThermoMgr",
+        "To be removed after Cantera 3.1. Replaceable by 'setThermo'.");
+    setThermo(thermo);
```

Comment:
As a `protected` method, `setThermo` isn't really a replacement -- the replacement is to specify the phase when constructing the reactor. Likewise for `setKineticsMgr`.